demo/.gitignore

```diff
@@ -32,3 +32,6 @@ yarn-error.log*
 
 # vercel
 .vercel
+
+# package lock file
+package-lock.json
```

Comment:
can you revert this? currently using yarn for the project and while i'm not opposed to change to npm with good reason, this isn't the right PR for it

Reply:
It was meant for the `.gitignore` on the root but yeah, sure, I'll use yarn